Add spacy-span-analyzer to debug data (#10668)
explosion/spaCy: website/docs/api/cli.md

Analyze, debug and validate your training and development data. Get useful
stats, and find problems like invalid entity annotations, cyclic dependencies,

> low data labels and more. 

Reviewer: ```suggestion
low data labels and more.
```

</Infobox>

<Infobox title="Notes on span characteristics" emoji="💡">

> If your pipeline contains a `"spancat"` component, then this command will also

Reviewer: ```suggestion
If your pipeline contains a `spancat` component, then this command will also
```

report span characteristics such as the average span length and the span (or
span boundary) distinctiveness. The distinctiveness measure shows how different
the tokens are with respect to the rest of the corpus using the KL-divergence of

> the token distributions. To learn more, you can check Papay et al's work on

Reviewer: ```suggestion
the token distributions. To learn more, you can check Papay et al.'s work on
```

> 
> <Infobox title="Notes on span characteristics" emoji="💡">
> 
> If your pipeline contains a `spancat` component, then this command will also
> report span characteristics such as the average span length and the span (or
> span boundary) distinctiveness. The distinctiveness measure shows how different
> the tokens are with respect to the rest of the corpus using the KL-divergence of
> the token distributions. To learn more, you can check Papay et al.'s work on
> [*Dissecting Span Identification Tasks with Performance Prediction* (EMNLP
> 2020)](https://aclanthology.org/2020.emnlp-main.396/).
> 
> </Infobox>
> 
> 
> 

Reviewer: ```suggestion
<Infobox title="Notes on span characteristics" emoji="💡">

If your pipeline contains a `spancat` component, then this command will also
report span characteristics such as the average span length and the span (or
span boundary) distinctiveness. The distinctiveness measure shows how different
the tokens are with respect to the rest of the corpus using the KL-divergence of
the token distributions. To learn more, you can check out Papay et al.'s work on
[*Dissecting Span Identification Tasks with Performance Prediction* (EMNLP
2020)](https://aclanthology.org/2020.emnlp-main.396/).

</Infobox>

```
